src/data/queries/doiQuery.ts

```diff
@@ -8,25 +8,33 @@ import ISO6391 from 'iso-639-1'
 export async function fetchDoi(id: string) {
   try {
     const options = { method: 'GET', headers: { accept: 'application/vnd.api+json' } };
-    const res = await fetch(`${process.env.NEXT_PUBLIC_API_URL}/dois/${id}?affiliation=false&publisher=true`, options)
+    const searchParams = new URLSearchParams({
+      query: 'uid:' + id,
+      include: 'client',
+      affiliation: 'false',
+      publisher: 'true',
+      'disable-facets': 'true',
+      include_other_registration_agencies: 'true'
+    })
+
+    const res = await fetch(`${process.env.NEXT_PUBLIC_API_URL}/dois?${searchParams.toString()}`, options)
     const json = await res.json()
-    const attrs = json.data.attributes
-
-    const repRes = await fetch(`${process.env.NEXT_PUBLIC_API_URL}/clients/${json.data.relationships.client.data.id}`, options)
-    const repJson = await repRes.json()
+    const work = json.data[0]
+    const attrs = work.attributes
+    const repo = json.included[0]
```

Comment:
Rather than just taking the first included object, it is might be good to iterate through and select the first one that has an id == work.relationships.client.data.id
If that even exists.  (For OtherDois it probably wont)